lib/tasks/build.js

```diff
@@ -3,13 +3,12 @@
 const path = require('path');
 const merge = require('merge-stream');
 const BowerPlugin = require('bower-webpack-plugin');
-const webpack = require('webpack-stream');
+const webpack = require('webpack');
+const webpackStream = require('webpack-stream');
```

Comment:
Well, if you have it installed locally it won't error, and I'm not aware of
how to do a prod build test. In ci it should have failed

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.